Database/Entities/MessageEntity.cs

```diff
@@ -21,7 +21,7 @@ public MessageEntity(string content, int senderId, int chatId, DateTime createdA
 		public DateTime CreatedAt { get; set; }
 
 		[ForeignKey(nameof(ChatId))]
-		public virtual ChatEntity? Chat { get; set; }
+		public ChatEntity Chat { get; set; } = new ChatEntity();
```

Comment:
I believe this is bad practice since it's misleading and you actually might not have a related chat. Better to be null by default, otherwise potential bugs might occur.